app/src/hid.c

```diff
@@ -10,7 +10,7 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 #include <zmk/hid.h>
 #include <dt-bindings/zmk/modifiers.h>
 
-static struct zmk_hid_keypad_report kp_report = {
+static struct zmk_hid_keyboard_report kp_report = {
```

Comment:
Thoughts on renaming the variable too, e.g. `kb_report`?

Reply:
Good idea.  It's slightly outside the scope of this PR so I'll follow up with another after this is merged.